frontend/templates/views/partials/version-toggle.j2

```diff
@@ -11,4 +11,14 @@
   </select>
 </div>
 {% endif %}
+
+<div>
+{% do doc.styles.addCssFile('css/components/atoms/pill.css') %}
+{% if doc.experimental %}
+  <div class="ap-a-pill ap-a-pill-small ap-a-pill-meta">Experimental</div>
```

Comment:
Pass both "Experimental" and "Deprecated" though our translation function like: `_('Experimental')`